resources/views/components/argument-card/card.blade.php

```diff
@@ -8,7 +8,21 @@
 
     <div class="grid gap-2 md:gap-4 w-full">
         @if(!$readonly && $isEditing?->is($argument))
-            <textarea wire:model="body" class="border-gray-200 rounded-lg" rows="5"></textarea>
+            <div>
+                <div wire:ignore>
+                <textarea id="argument-editor" x-init="
+                test = new EasyMDE({
```

Comment:
Not sure I like the approach. I happy to look at some suggestions to clean this a bit.
Initially I tried to have this as a dedicated component which can be reused, but I wasn't able to figure out how this `wire:model` really works.